inference-engine/src/preprocessing/cpu_x86_avx2/ie_preprocess_gapi_kernels_avx2.cpp

```diff
@@ -147,75 +468,35 @@ void calcRowLinear_8UC_Impl(std::array<std::array<uint8_t*, 4>, chanNum> &dst,
     const int shift = (half_nlanes / 4);
 
     if (4 == lpi) {
-        GAPI_DbgAssert(inSz.width >= half_nlanes);
+        GAPI_DbgAssert(inSz.width*chanNum >= half_nlanes);
 
         v_uint8 shuf_mask1 = v_setr_s8(0, 8,  4, 12, 1, 9,  5, 13,
                                        2, 10, 6, 14, 3, 11, 7, 15,
                                        0, 8,  4, 12, 1, 9,  5, 13,
                                        2, 10, 6, 14, 3, 11, 7, 15);
```

Comment:
I believe this mask can be moved under horizontalPass_lpi4_8UC1

Reply:
done